packages/baseai/src/dev/llms/utils.ts

```diff
@@ -30,11 +32,12 @@ export function applyJsonModeIfEnabled(modelParams: ModelParams, pipe: any) {
 
 export function applyJsonModeIfEnabledForGoogle(
 	transformedRequestParams: any,
-	pipe: any
+	pipe: Pipe
 ) {
+	const currentModel = pipe.model.split(':')[1];
```

Comment:
Same. I think `modelParams.model` will already have this name

Reply:
`modelParams` is not being passed to this function